### DISCUSSION TITLE
Any HuggingFace employee use microcontrollers?

### DISCUSSION THREAD

**Discussion by hpssjellis** [huggingface/huggingface.js#99] [Ideas]
@radames @coyotte508 any idea if anyone at HuggingFace is a microcontroller expert. I have a few ideas.
![GIF-230224_074433](https://user-images.githubusercontent.com/5605614/225045111-06cba67e-4589-4015-9da4-14844648d0a7.gif)

**Comment by coyotte508** [comment]
you should join our discord: https://discuss.huggingface.co/t/join-the-hugging-face-discord/11263

There are tons of people there with different specialties

**Reply by hpssjellis** [reply]
@coyotte508 

> you should join our discord: https://discuss.huggingface.co/t/join-the-hugging-face-discord/11263
 

I joined discord, any chance of getting someone to make a JavaScript channel?

**Reply by radames** [reply]
hi @hpssjellis let's start a thread on [#general ](https://discord.com/channels/879548962464493619/879548962464493622) and see if there is more interested people, to start a new channel. You can also explore [#today-im-learning](https://discord.com/channels/879548962464493619/898619964095860757) and [#looking-for-collaborators](https://discord.com/channels/879548962464493619/897390653456125995)
